db/data/custom_data.rb

```diff
@@ -0,0 +1,31 @@
+# Data for Address
+WARDS = [ 'Adachi', 'Arakawa', 'Bunkyo', 'Chiyoda', 'Chuo', 'Edogawa', 'Itabashi', 'Katsushika', 'Kita', 'Koto', 'Meguro', 'Minato', 'Nakano', 'Nerima', 'Ota', 'Setagaya', 'Shibuya', 'Shinagawa', 'Shinjuku', 'Suginami', 'Sumida', 'Taito', 'Toshima' ]
```

Comment:
can you please add also wards/cities of prefectures surrounding tokyo? (kanagawa, chiba...)

Reply:
on it